lib/services/local/access_list_test.go

```diff
@@ -125,6 +125,145 @@ func TestAccessListCRUD(t *testing.T) {
 	require.Empty(t, out)
 }
 
+func TestAccessListMembersCRUD(t *testing.T) {
```

Comment:
Thanks, this is better.

I won't hold you on this, but I still think these do too much. In the future I'd consider splitting tests a bit more.

Reply:
Yeah, agreed. I tried to split it out for a bit but I realized it was going to take a pretty substantial amount of time, so I punted here.